Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -45,14 +45,17 @@ sed -i.bak \
     -e "s/\(.*\):.*# namespace:cluster/\1: $ROOK_CLUSTER_NAMESPACE # namespace:cluster/g" \
     -e "s/\(.*serviceaccount\):.*:\(.*\) # serviceaccount:namespace:operator/\1:$ROOK_OPERATOR_NAMESPACE:\2 # serviceaccount:namespace:operator/g" \
     -e "s/\(.*serviceaccount\):.*:\(.*\) # serviceaccount:namespace:cluster/\1:$ROOK_CLUSTER_NAMESPACE:\2 # serviceaccount:namespace:cluster/g" \
-    -e "s/\(.*\): [-_A-Za-z0-9]*\.\(.*\) # driver:namespace:operator/\1: $ROOK_OPERATOR_NAMESPACE.\2 # driver:namespace:operator/g" \
```

Comment:
revert?

Reply:
Ah I see now thanks. I had overlooked this tag. The rbd storage class was missing this tag completely. Actually I wonder if we should keep that tag instead of using the new tag I suggested. Then if anyone has existing scripts, they won't need to update to the new tag. The new tag name seems better since it might not be related to the namespace now. So I'm fine with the new tag, but could go either way. Thoughts?

Reply:
That is correct, but the new tag makes sense to me as it doesn't talk about namespace and we do support multiple ways now.